src/main/java/net/sf/jabref/gui/importer/fetcher/ACMPortalFetcher.java

```diff
@@ -157,19 +155,11 @@ public boolean processQueryGetPreview(String query, FetcherPreviewDialog preview
 
             return true;
 
-        } catch (MalformedURLException e) {
-            LOGGER.warn("Problem with ACM fetcher URL", e);
-        } catch (ConnectException e) {
-            status.showMessage(Localization.lang("Could not connect to %0", getTitle()),
-                    Localization.lang("Search %0", getTitle()), JOptionPane.ERROR_MESSAGE);
-            LOGGER.warn("Problem with ACM connection", e);
         } catch (IOException e) {
-            status.showMessage(e.getMessage(),
-                    Localization.lang("Search %0", getTitle()), JOptionPane.ERROR_MESSAGE);
-            LOGGER.warn("Problem with ACM Portal", e);
+            LOGGER.error("Error while fetching from " + getTitle(), e);
+            preview.showErrorMessage(this.getTitle(), e.getLocalizedMessage());
+            return false;
         }
```

Comment:
I like to return immediately in general so I forget that branch.
If someone now breaks out of the try-catch (which shouldn't be done) he is forced to deal with it later down.